mapparser.py

```diff
@@ -190,7 +196,11 @@ def get_same_room_objects(self, player: player.Player) -> list:
                             (group != "" and group == player.group) or \
                             (name != "???" and name == player.name):
                             objX, objY = objX % 32 // 4, objY % 32 // 4
-                            objects.append((obj.attrib.get("name", "???"), objX, objY, obj.attrib.get("class", "")))
+                            width = int(obj.attrib.get("width", 0)) // 4
+                            height = int(obj.attrib.get("height", 0)) // 4
+                            layer = objectgroup.attrib["name"]
+                            layer = 0 if layer == "нижний" else 1 if layer == "средний" else 2
+                            objects.append((obj.attrib.get("name", "???"), objX, objY, obj.attrib.get("class", ""), width, height, layer))
```

Comment:
it would be better to create a dataclass for these objects to avoid magic numbers